Makefile

```diff
@@ -72,6 +72,10 @@ dm-syncer:
 debug-tools:
 	$(GOBUILD) -ldflags '$(LDFLAGS)' -o bin/binlog-event-blackhole ./debug-tools/binlog-event-blackhole
 
+dm-portal-frontend:
+	# TODO: build frontend
```

Comment:
@baurine could you please add some command to build frontend?

Reply:
```
```

dm-portal-frontend:
	cd ./dm/portal/frontend
	yarn
	yarn build
	cd -
	statik -src=./dm/portal/frontend/build -dest=./dm/portal/
```

run `yarn && yarn build` in the frontend folder will build the frontend app.

Reply:
OK, thanks